app/helpers/application_helper.rb

```diff
@@ -15,4 +15,8 @@ def application_meta_tags
     ].join("\n").html_safe
   end
 
+  def favicon_file_name_suffix
+    ENV['FAVICON_FILE_NAME_SUFFIX']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.